examples/webgl_materials_modified.html

```diff
@@ -57,8 +57,8 @@
 				document.body.appendChild( renderer.domElement );
 
 				var controls = new OrbitControls( camera, renderer.domElement );
-				controls.minDistance = 1000;
-				controls.maxDistance = 5000;
+				controls.minDistance = 0;
+				controls.maxDistance = 90;
```

Comment:
Never use zero.
```js
controls.minDistance = 10;
controls.maxDistance = 50;
```
Or, instead, simply
```js
controls.enableZoom = false;
```

Reply:
I changed the ranges but left the zoom 👍